### DISCUSSION TITLE
OneDrive remeaning space calculated incorrectly

### DISCUSSION THREAD

**Discussion by victoriusnet** [abraunegg/onedrive#2741] [Q&A]
Hey OneDrive experts,

I have bumped into an issue months ago which could not be resolved even a Microsoft engineer (he said it's an expected behaviour lol):

I'm using this fantastic tool to upload my backups and I used to reach my 1TB limit when I got an email that my storage is full. 
Nothing special, I have manually deleted some old backups and its resolved. (I'm aware that 2nd stage recycle bin exists as well).

For some still unknown reason I have stopped to receive email notifications but the OneDrive client stopped uploading due no free space remained. No email at all.

I have deleted some files again, but this time I was not getting back as much free space as files I have deleted:
![onedrive-free-space](https://github.com/abraunegg/onedrive/assets/48323740/2edd119c-6079-4415-bc4a-26c764a3ebda)

If you take a look at my screenshot the remaining space in storage metrics is invalid. I'm missing around 200GB. Whenever my storge is full I only know as OneDrive client fails to upload but storage metrics reports ~200GB free space and not sending an email to me.

OneDrive client and GraphAPI are reporting the valid numbers:

"quota": {
        "deleted": 0,
        "remaining": 389192683040,
        "state": "normal",
        "total": 1099511627776,
        "used": 710318944736
    }
    
Do anyone has some tips what can I do to let storage metrics the same value as GraphAPI and be able to get email reports about the disk space? I think this two things has a connection.
    
Thank you!

**Comment by abraunegg** [comment]
@victoriusnet 
The `% of parent` being 100% does not add up here for me, and it looks like you may have allocated a 'quota' (how much space can Documents use) vs 'just use all the space' ... so whilst your email triggers are for 'free space' it is not factoring in '% quota used'

I am not a SharePoint expert - but I would be looking at or trying to find settings related to Quota, at this level and at the parent level where 'Documents' is located.
